app/gui2/src/components/ToggleIcon.vue

```diff
@@ -0,0 +1,20 @@
+<script setup lang="ts">
+import SvgIcon from '@/components/SvgIcon.vue'
+import { ref } from 'vue'
+
+const props = defineProps<{ icon: String; modelValue?: Boolean }>()
```

Comment:
use `string` instead of `String` and `boolean` instead of `Boolean` - the capitalized variants are interfaces, not the primitives (although the primitives do satisfy the interfaces)

Reply:
btw, is there a usecase in which `modelValue` shouldn't be provided?